examples/jsm/renderers/nodes/accessors/CameraNode.js

```diff
@@ -69,13 +69,13 @@ class CameraNode extends Node {
 
 			if ( scope === CameraNode.PROJECTION || scope === CameraNode.VIEW ) {
 
-				if ( !inputNode || !inputNode.isMatrix4Node ) {
+				if ( !inputNode || !inputNode.isMatrix4Node !== true ) {
```

Comment:
This still does not look right. Notice that a leading `!`s are hard to read. Hence, we usually check for a specific value for type:
```js
if ( inputNode === undefined  || inputNode.isMatrix4Node !== true ) {
```